csharp/Samples/Microsoft.Spark.CSharp/samplesusage.md

```diff
@@ -0,0 +1,50 @@
+SparkCLRSamples.exe supports following options:
+   
```

Comment:
LGTM

Consider adding bullets to the options and also format the arg names and values (for example, argument name like --temp or spark.local.dir could be in bold and value like <TEMP_DIR> could be in italics)